src/uniswap/incentives.ts

```diff
@@ -191,30 +203,38 @@ export const incentivesDistribution = (
    */
   const incentivesDecimals = 14n;
 
+  /*
+   * Computes the amount of incentives that should be received for the specified amount of incentive
+   * tokens.  Uses `incentivesDecimals` as precision when computing fixed decimal numbers.
+   */
+  const incentivesForTokens = (tokens: bigint): number => {
+    const asFixedDecimal =
```

Comment:
This is probably OK, as there seems to be no way to avoid inaccuracies when we convert 256 bit values into floating point integers with only 53 bits of mantissa.
Inaccuracies will occur first in the lowest bits, which represent digits 14 decimal places after the point.
Not sure what is an alternative here.